build.cmd

```diff
@@ -24,10 +24,10 @@ if not exist %NUGET_LOCAL_DIR%\NuGet.exe (
 )
 
 :: restore packages
-%NUGET_LOCAL_DIR%\NuGet.exe restore .\packages.config -PackagesDirectory ./src/packages -MSBuildVersion 15 -Verbosity quiet
+%NUGET_LOCAL_DIR%\NuGet.exe restore .\packages.config -PackagesDirectory ./build-packages -MSBuildVersion 15 -Verbosity quiet
```

Comment:
The thought behind this was that this folder is now only for the build dependencies, since everything else is now using PackageReferences. That was why I moved it into the repo root instead of being in `/src`. Once it was there, after running the build script, we'd end up with both a `packages` folder and a `packaging` folder. I didn't care for how similar those looked, so I added `build` to the name of the folder.

The next step after merging this is to use the built-in packaging capabilities, as I've done [here](https://github.com/Particular/NServiceBus.Transport.Msmq/pull/3/commits/b402c6da5d4ce23754052119c63359fbd46ab198) for MSMQ, so once that is done, the `packaging` folder would be gone.

While I still think there is some value in indicating the folder it just for build script dependencies, I'm more OK with just calling it `packages` at that point.